dot/parachain/available_data_fetching.go

```diff
@@ -57,6 +57,7 @@ type AvailableData struct {
 	ValidationData PersistedValidationData `scale:"2"`
 }
 
+// TODO: remove this type, copied to dot/parachain/types
```

Comment:
is it a big refactor removing this type in this PR?

Reply:
Nope, I removed it.